### PR TITLE
Feature/eu and uk welcome banner

### PR DESCRIPTION
**Pull request by alamin-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/layout/welcome_banner.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+const countriesBanner = {
+    uk: ukbanner,
+    eu: eubanner,
+    euMobile: eubannerMobile,
+    ukMobile: ukbannerMobile,
+}
+
+const BannerWrapper = styled.section<{ country: string }>`
+    background-color: #3c77ae;
```
Comment (prince-deriv) [inline src/components/layout/welcome_banner.tsx:19]: Do we have this color in our theme? If not can we add this?

Comment (alamin-deriv) [reply src/components/layout/welcome_banner.tsx:19]: okay, let me checkout

Comment (alamin-deriv) [reply src/components/layout/welcome_banner.tsx:19]: No we don't have it actually.

**Comment by sean-binary** [comment]
For the close icon, can we please use existing SVGs for it?

**Comment by alamin-deriv** [comment]
> For the close icon, can we please use existing SVGs for it?

Okay sure, let me look into it.

**Comment by sean-binary** [comment]
Also you might want to checkout the overlap that is happening in `id` for certain screen widths. @alamin-deriv

**Comment by alamin-deriv** [comment]
> For the close icon, can we please use existing SVGs for it?

@sean-binary but the SVG icon is gray not white
Screenshot 2022-02-28 at 6 46 14 AM

**Comment by alamin-deriv** [comment]
> Also you might want to checkout the overlap that is happening in `id` for certain screen widths. @alamin-deriv

hello, @sean-binary I didn't understand the part  ` is happening in id` can you please explain more, thank you.

**Comment by sean-binary** [comment]
Sure, the svg is fine 👍 

> > Also you might want to checkout the overlap that is happening in `id` for certain screen widths. @alamin-deriv
> 
> hello, @sean-binary I didn't understand the part ` is happening in id` can you please explain more, thank you.

image
Here you go.

**Comment by alamin-deriv** [comment]
> Sure, the svg is fine 👍
> 
> > > Also you might want to checkout the overlap that is happening in `id` for certain screen widths. @alamin-deriv
> > 
> > 
> > hello, @sean-binary I didn't understand the part ` is happening in id` can you please explain more, thank you.
> 
> image Here you go.

oh okay let me fix that

**Comment by alamin-deriv** [comment]
> Sure, the svg is fine 👍
> 
> > > Also you might want to checkout the overlap that is happening in `id` for certain screen widths. @alamin-deriv
> > 
> > 
> > hello, @sean-binary I didn't understand the part ` is happening in id` can you please explain more, thank you.
> 
> image Here you go.

@sean-binary Fixed

File: src/components/layout/welcome_banner.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+}
+type currentProbs = {
+    base: baseProbs
+}
+type CFDWarningRefProbs = {
+    current: currentProbs
+}
+type WelcomeBannerProbs = {
+    CFDWarningRef: CFDWarningRefProbs
+}
```
Comment (sean-binary) [inline src/components/layout/welcome_banner.tsx:87]: Should be `Props` instead of `Probs` 😆

File: src/components/layout/welcome_banner.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+export const WelcomeBanner = ({ CFDWarningRef }: WelcomeBannerProbs) => {
+    const [is_welcome_banner_dismissed, setWelcomeBanner] = useState(null)
```
Comment (sean-binary) [inline src/components/layout/welcome_banner.tsx:90]:
```suggestion
    const [is_welcome_banner_dismissed, setWelcomeBannerDismissed] = useState(null)
```

**Comment by sean-binary** [comment]
Other than those, everything else looks good! 👍

File: src/components/layout/welcome_banner.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        width: 100%;
+        padding: 16px;
+        height: fit-content;
+        margin-bottom: 0;
+        bottom: 86px;
+    }
+
+    @media (max-width: 540px) {
+        bottom: 112px;
+    }
```
[the remark this answers is not in the document]
Comment (alamin-deriv) [reply src/components/layout/welcome_banner.tsx:50]: It works fine on mobile

**Review by sean-binary — APPROVED** [review]

**Pull request closed without merging**